vocabularies/gsq-dataset-themes.ttl

```diff
@@ -0,0 +1,100 @@
+    skos:inScheme :conceptScheme ;
+    skos:prefLabel "Administrative data"@en ;
+    skos:topConceptOf :conceptScheme .
+
+:geoscience-data a skos:Concept ;
+    skos:definition "Primary descriptive data defining the geological attributes of a feature of interest."@en ;
+    skos:inScheme :conceptScheme ;
+    skos:prefLabel "Geoscience data"@en ;
+    skos:topConceptOf :conceptScheme .
+
+:geoscience-project-data a skos:Concept ;
+    skos:definition "Data and media describing project information such as activities performed, planning and approvals for the collection and analysis of primary geoscience data."@en ;
+    skos:inScheme :conceptScheme ;
+    skos:prefLabel "Geoscience project data"@en ;
+    skos:topConceptOf :conceptScheme .
+
+:other a skos:Concept ;
+    skos:definition "Any other metadata, data, or information that is not described by any of the other GSQ Dataset Themes."@en ;
+    skos:inScheme :conceptScheme ;
+    skos:prefLabel "Other"@en ;
+    skos:topConceptOf :conceptScheme .
+
+:operational-data a skos:Concept ;
+    skos:definition "Data describing the operations of mining, drilling, processing, and sales operations e.g. borehole drilling, pipelines, processing facilities."@en ;
+    skos:inScheme :conceptScheme ;
+    skos:prefLabel "Operational data"@en ;
+    skos:topConceptOf :conceptScheme .
+
+:physical-samples a skos:Concept ;
+    skos:definition "Data and metadata pertaining to physical samples taken during geoscience data acquisition e.g. drill core, rock chips, rock samples from outcrop, soil, sediment, water, drilling fluids, oil, gas etc."@en ;
+    skos:inScheme :conceptScheme ;
+    skos:prefLabel "Physical samples"@en ;
+    skos:topConceptOf :conceptScheme .
+
+:publications a skos:Concept ;
+    skos:definition "Publications that describe the geology, resource potential, or other areas of departmental interest such as books, journals, periodicals, presentations, posters, academic publications, or multimedia products."@en ;
+    skos:inScheme :conceptScheme ;
+    skos:prefLabel "Publications"@en ;
+    skos:topConceptOf :conceptScheme .
+
+:reports a skos:Concept ;
+    skos:definition "Reports lodged as part of an approved study or activity, commonly for the purpose of compliance with permit grant and retention conditions. These reports may be permit based such as a resource authority annual report, activity based such as well completion reports, or non-tenure reports such as departmental survey reports."@en ;
+    skos:inScheme :conceptScheme ;
+    skos:prefLabel "Reports"@en ;
+    skos:topConceptOf :conceptScheme .
+
+:resource-statistics a skos:Concept ;
+    skos:definition "Statistical data and reports submitted to the department or collected/generated by the department, particularly production and sales quantities."@en ;

```

Comment:
Minor change to 'Statistical data and reports submitted to the department or collated/generated by the department, particularly production, sales and reserve quantities'

Reply:
changed.